src/main/java/seedu/eduke8/exception/ExceptionMessages.java

```diff
@@ -0,0 +1,18 @@
+package seedu.eduke8.exception;
+
+public class ExceptionMessages {
+
+    public static final String ERROR_STORAGE_FAIL = "Oh no! An error has occurred when accessing the file."
+            + "Please check again!";
+    public static final String ERROR_UNKNOWN = "Oh no! An unknown error has occurred.";
+    public static final String ERROR_UNRECOGNIZED_COMMAND = "Oops! The command cannot be understood.\n"
+            + "Please use the \"help\" to get the list of available commands. ";
```

Comment:
Do you think this might be better if phrased as "Please enter in 'help' to get the list of available commands." ?

Reply:
Ok done!